wordpress/wp-content/plugins/cds-base/classes/Modules/Checklists/js/meta-box.js

```diff
@@ -0,0 +1,229 @@
+jQuery(document).ready(
+    function() {
+        /* Assigning vars */
+        var ppc_publish_flag = 0;
+        var ppc_checkboxes = jQuery('.ppc_checkboxes[type="checkbox"]');
+        var ppc_checkboxes_length = jQuery('.ppc_checkboxes[type="checkbox"]').length;
+        var countCheckedppc_checkboxes = ppc_checkboxes.filter(':checked').length;
+        var ppc_percentage_completed = (countCheckedppc_checkboxes / ppc_checkboxes_length) * 100;
+        jQuery('.ppc-percentage').attr('style', 'width:' + ppc_percentage_completed + '%');
+        jQuery(".ppc-percentage-value").html(Math.round(ppc_percentage_completed) + "%");
+
+
+        // .editor-post-publish-panel__toggle   = "publish" button for unpublished posts 
+        // .editor-post-publish-button          = "update" button for already-published posts
+
+        //function to be executed when the itemlist changes.
+        
+        var ppc_checkbox_function = function() {
+            var ppc_checkboxes = jQuery('.ppc_checkboxes[type="checkbox"]');
+            var ppc_checkboxes_length = jQuery('.ppc_checkboxes[type="checkbox"]').length;
```

Comment:
Do we need these if they are assigned above

Reply:
No!